packages/core/src/lib/icon/icons.ts

```diff
@@ -13,7 +13,7 @@ export type IconPath = Path | CustomIcon[];
  * Keys should match MDI name
  * e.g. 'account-multiple' for MDI.mdiAccountMultiple
  */
-export const paths: Record<string, IconPath> = {
```

Comment:
This `Record<string, IconPath>` overrode the `as const` below and broke strict icon name type-checking, setting `IconName` to `string` rather than the literal keys of this object

Reply:
Nice catch!